packages/jest-config/src/__tests__/normalize.test.ts

```diff
@@ -9,7 +9,6 @@
 import {createHash} from 'crypto';
 import path from 'path';
 import semver = require('semver');
-import stripAnsi from 'strip-ansi';
```

Comment:
The dev dependency seems to be unnecessary as well, but please double check:

https://github.com/facebook/jest/blob/201901f3e75d0a08bc5f08dc1a4b9fd8ea4c67b3/packages/jest-config/package.json#L58

Reply:
you're correct, only used here

Reply:
yeah let me see